docs/spherical-video-rfc.md

```diff
@@ -89,7 +111,7 @@ The default initial viewport is set such that the frame center occurs at the vie
     o  - the image center for a pitch of 45 and a heading of 90
     >  - the up vector for a rotation of 90 degrees.
 
-### Local Metadata
+### Local Metadata<a name="LocalMetadata"/a>
```

Comment:
above you have the <a> link for InitialView before the ### part

does it make a different? is it possible to be consistent?